build.sbt

```diff
@@ -467,7 +470,7 @@ lazy val compiler = configureAsSubproject(project)
       "/project/description" -> <description>Compiler for the Scala Programming Language</description>,
       "/project/packaging" -> <packaging>jar</packaging>
     ),
-    apiURL := None,
+    apiURL := Some(url(s"https://www.scala-lang.org/api/${versionProperties.value.mavenVersion}/scala-${thisProject.value.id}/")),
```

Comment:
Maybe this should just be the literal name of the project, e.g.`scala-compiler`, instead of `scala-${thisProject.value.id}`?